src/components/UniprotInput.vue

```diff
@@ -86,6 +88,44 @@ export default Vue.extend({
   created() {
     this.debouncedQuery = debounce(this.query, 500);
   },
+  watch: {
+    forceSearchQuery: {
+      // Watcher needs to be immediate to trigger when copy-paste creates
+      // new rows with forceSearchQuery already set
+      immediate: true,
+      handler() {
+        if (!this.forceSearchQuery) {
+          return;
+        }
+        this.searchQuery = this.forceSearchQuery;
+      }
+    },
+    searchQuery: {
+      // Watcher needs to be immediate to trigger when copy-paste creates
+      // new rows with forceSearchQuery already set
+      immediate: true,
+      handler() {
+        // Reset state
+        this.protein = null;
+        this.requestError = false;
+        this.$emit("change", this.protein);
```

Comment:
I would emit `change` event only if exact match was found, then in `NewExperiment` component, `protein` field would still have property `_pastedText` and error message would be `No match was found` instead of `Required`

Reply:
That would be better, but then how do we signal that the previous match object was removed, when the user is typing manually? If I remove this event, if I first type a valid query, and then change it, then there is no message:

![image](https://user-images.githubusercontent.com/64745/67583367-b9017880-f74b-11e9-8d1a-6a2327773623.png)

Reply:
Maybe if you emit null in `if (!this.searchQuery)` and on `404`

Reply:
*not on 404

Reply:
No, wait, nevermind